presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -59,6 +59,46 @@ General Aggregate Functions
 
     Returns the geometric mean of all input values.
 
+.. function:: grouping(col1, ..., colN) -> integer
+
+    Returns a bit set converted to decimal, indicating which columns are present in a grouping. The function
+    must be used in conjunction with ``GROUPING SETS``, ``ROLLUP`` or ``CUBE`` and its arguments must
+    match exactly the columns referenced in the corresponding ``GROUPING SETS``, ``ROLLUP`` or ``CUBE``.
+
+    To compute the resulting bit set for a particular row, bits are assigned to the argument columns with
+    the rightmost column being the least significant bit. For a given grouping, a bit is set to 0 if the
```

Comment:
This seems backwards. According to the ANSI spec:

> The result of such an invocation is 1 (one) in the case of a row whose values are the results of aggregation over that <column reference> during the execution of a grouped query containing CUBE, ROLLUP, or GROUPING SETS, and 0 (zero) otherwise.